WordPress/src/main/java/org/wordpress/android/ui/jetpack/scan/history/ScanHistoryFragment.kt

```diff
@@ -1,34 +1,44 @@
 package org.wordpress.android.ui.jetpack.scan.history
 
 import android.os.Bundle
+import android.view.MenuItem
 import android.view.View
+import androidx.appcompat.app.AppCompatActivity
 import androidx.fragment.app.Fragment
 import androidx.lifecycle.ViewModelProvider
+import androidx.viewpager2.adapter.FragmentStateAdapter
+import com.google.android.material.tabs.TabLayoutMediator
+import kotlinx.android.synthetic.main.scan_history_fragment.*
 import org.wordpress.android.R
+import org.wordpress.android.R.string
```

Comment:
Nitpick 🔍: We can remove this import.

Reply:
Fixed in https://github.com/wordpress-mobile/WordPress-Android/pull/13757/commits/5dfb89cf2de980af3dace30bc8d4a96994322363